<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_Mike_on">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 52, 197);
font: 75 13pt &quot;Ubuntu Mono&quot;;</string>
   </property>
   <property name="text">
    <string>Mic On</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_User">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>사용자 명</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Output">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>230</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 255, 96);
font: 75 13pt &quot;Ubuntu Mono&quot;;</string>
   </property>
   <property name="text">
    <string>음성 출력</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_Model">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>F</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Default</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="User_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>마이크에서 입력받은 말을 텍스트로 출력</string>
   </property>
  </widget>
  <widget class="QLabel" name="Model_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>마이크에서 입력받은 말을 텍스트로 출력</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
